\chapter{Conclusion}\label{conclusion}

In Chapter \ref{ch:tb}, I described my work investigating the innate immune
response to \emph{Mycobacterium tuberculosis} (MTB) \citep{Blischak2015}. It is
known that the innate immune response is important for fighting MTB
infections. Alveolar macrophages are the primary target of MTB, and they
initiate the formation of granulomas to sequester MTB. Furthermore, vaccines
against TB have had limited efficacy. To identify human genes which are
important for the response to MTB infection, we isolated macrophages from 6
healthy donors, infected them with MTB and other bacteria, and measured
genome-wide gene expression levels using RNA-seq at 4, 18, and 48 hours
post-infection.

Previous studies had identified the genes which
are differentially expressed upon infection with MTB, and some have
even compared the differences between the reponse to strains of MTB
that vary in their virulence. The first novelty of our study was to
include other bacteria in the infection experiments. Specifically, we
included the following Mycobacteria: two strains of virulent MTB,
avirulent (heat-inavtivated) MTB, the attentuated \emph{Mycobacterium
  bovis} (used as a vaccine), and the avirulent \emph{Mycobacterium
  smegmatis}. The non-mycobacteria species we included were
\emph{Yersinia pseudotuberculosis}, \emph{Salmonella typhimurium}, and
\emph{Staphylococcus epidermidis}. This allowed us to distinguish
between the innate immune response to MTB versus other virulent
bacteria, MTB versus avirulent mycobacteria, amd MTB versus deceased
MTB.

This novel study design comparing many bacterial infections to isolate
the innate immune respone to MTB also posed analytical
challenges. Standard differential expression analyses (or in general
any large scale testing of thousands or more genomic features) are
well-suited for experiments with a few conditions. For example, the
most common approach is to perform pairwise differential expression
tests and then overlap the lists of differentially expressed
genes. These results are always biased by incomplete power. Because
hypothesis testing uses an arbitrary p-value threshold to determine
statistical significance, a gene with a p-value below this threshold
for one comparison but a p-value slightly above this threshold for a
separate comparison will be classified as specific to the first when
in reality the gene is behaving similarly in both. As the number of
pairwise comparisons increases, the problem of incomplete power is
exacerbated, i.e. a gene is more likely to be statistically
significant for some susbset of comparisons. This increase in
comparisons also decreases the ability to interpret the results. A
3-way Venn diagram (and perhaps a 4- or 5-way) can be interpreted, but
this approach breaks down with additional comparisons.

Another approach would be to directly compare the effect of infection between
two different groups of bacteria, e.g. compare the mycobacteria versus the
non-mycobacteria or virulent versus non-virulent bacteria. The advantage of this
approach is that it explicitly models the comparison and returns a p-value,
unlike the Venn diagram overlap approach. However, there are two main
downsides. First, statistical significance can be driven by outliers. For
example, most of the most significantly differentially expressed between
mycobacteria and non-mycobacteria were actually genes which were simply
differentially expressed in response to infection with \emph{S. typhimurium} and
\emph{S. epidermidis}. Second, this limits the results to the \emph{a priori}
ideas of the analyst and is not driven by the patterns in the actual data.

On the other end of the spectrum, a very data-driven approach would be to use a
clustering method such as hierarchical or k-means clustering. These multivariate
methods are able to find the patterns of gene expression in the data, both
expected and unexpected; however, since they are not accompanied by any formal
hypothesis test, it is difficult to interpret which clusters of co-expressed
genes are the most interesting to report.

Since none of the standard genomics approaches were adequate for properly
comparing 8 bacterial infections, I instead used a joint Bayesian model,
implemented in the software package Cormotif, to analyze the data. Conceptually,
Cormotif combines the clustering and pairwise testing approaches described
above. Just like the pairwise testing approach, the input to Cormotif are the
pairwise comparisons between each bacterial infection and the control
condition. However, to account for incomplete power, Cormotif models the gene
expression levels across all the pairwise comparisons to identify the main gene
expression patterns, conceptually similar to a clustering analysis.

The Cormotif results for my study were informative. Most of the genes were
either differentially expressed or not after infection with any of the
bacteria. The two most interesting patterns in regards to understanding the
innate immune response to MTB were MTB and Virulent.  The MTB pattern included
those genes which had a high posterior probability of being differentially
expressed to any of the MTB or closely related species and a medium posterior
probability of being differentially expressed to \emph{M. smegmatis}, the
nonvirulent mycobacteria. The Virulent pattern included genes which had a high
posterior probability of being differentially expressed in response to infection
with any of the bacteria except heat-inactivated MTB or BCG.

In terms of better understanding TB susceptibility, the main takeaway from this
study was the identification of hundreds of genes which are differentially
expressed in response specifically to infection with MTB and related species but
not other virulent bacteria. These genes are candidates for containing genetic
variants which affect TB susceptibility. Furthermore, these genes could be
targets for future functional studies of how the innate immune system fights MTB
and also could give context to future results from genetic and functional
genomics studies of MTB infection. More generally, our results are informative
to all future functional genomics studies. We were only able to confidently
isolate the effects of MTB infection by including multiple other bacterial
infections as comparison. Had we only infected the macrophages with MTB and
heat-inactivated MTB, we would have made multiple misclassifications. We would
have assigned differences between the two infections as specific to a live,
virulent MTB; however, these gene expression changes were also induced by other
live bacteria. Similarly, we would never have known that a subset of the genes
which were differentially expressed in response to both MTB and heat-inactivated
MTB were actually specific to mycobacteria in general. Not only was it important
to include multiple bacterial infections, but it was also critical to properly
analyze the results. Because the innate immune system is largely a general
response to infection, we expected most of the induced gene expression changes
to be similar across bacteria. Had we performed the straight-forward approach of
overlapping lists of differentially expressed genes from comparing the
individual infections to their controls, we would have had identified lots of
spurious differences in the innate immune response caused by incomplete
power. In contrast, by jointly modeling the data with Cormotif, we were able to
identify the shared gene expression patterns in response to related bacteria. In
support of the generality of this approach, the Cormotif approach was
successfully applied to distinguish the effects of vitamin D and bacterial
lipopolysaccharide on the innate immune response between individuals of
African-American and European-American ancestry (note: I was a co-author of the
study) \citep{Kariuki2016}.

It should be noted that this method also has its caveats. First, its strength of
sharing information across the pairwise comparisons can also be a negative
because it will not identify genes with unique expression patterns. While useful
for projects with the aim of broadly characterizing the genome-wide gene
expression patterns for a given phenomenon, it is not well-suited for
identifying outlier genes. Second, because the algorithm is not deterministic,
Cormotif must be run multiple times to obtain the model with the highest log
likelihood. Because of this added complexity, using Cormotif is more difficult
to implement than more standard differential expression approaches.

In Chapter \ref{ch:tb-suscept}, I described my work investigating the
role of gene regulation in the innate immune system on TB
susceptibility (not yet published). Specifically, in order to
investigate how the innate immune cells of susceptible individuals
function compared to those of resistant individuals, we collected
primary dendritic cells (DCs) from individuals that had recovered from
TB (i.e. susceptible) and individuals that tested positive for latent
MTB infections but had not developed TB (i.e. resistant). We infected
the DCs with MTB and performed RNA-seq on the infected and
non-infected cells.

There were 3 main conclusions from this work. First, the differences
in gene expression levels between resistant and susceptible
individuals were primarily present in the noninfected state and not 18
hours post-infection with MTB. This suggests that these gene
expression differences primarily affect the very early response to MTB
infection. Second, we discovered that the effect sizes measured in our
\emph{in vitro} experiment, whether comparing between resistant and
susceptible individuals or between the infected and noninfected
states, were negatively correlated with lower p-values from two genome
wide association studies (GWAS) of TB susceptibility. This suggests
that our \emph{in vitro} system is a useful model for investigating
the genetic basis of TB susceptibility. Third, we trained a classifier
based on the gene expression levels in the noninfected state. Using
the threshold required to obtain a 100\% false negative rate in the
training data, we found that 11\% of healthy individuals from an
independent study were predicted to be susceptible to TB, very close
to the estimated population average of 10\%. This suggests that
isolating innate immune cells and performing gene expression profiling
could be a feasible test for TB susceptibility.  The most obvious
extension of this work is to conduct a larger study with more
susceptible individuals. Our current results are only a
proof-of-principle. With a larger study, we could properly split the
data into training and test sets to assure that the model is not
overfitting the data. On the one hand, since we identified that the
gene expression differences are only present in the noninfected state
and that these are sufficient for the performance of the classifier,
this future study would be simplified by not having to perform the MTB
infections. On the other hand, collecting a large number of patient
samples is always difficult, and it is even worse when the individuals
are currently healthy and thus not regularly visiting the doctor like
those recovered from a past case of active TB. Hopefully these initial
successful results will provide the impetus for larger scale sample
collection.

Another fruitful direction for future experiments would be to further
investigate the role of \emph{CCL1} in the innate immune response to
MTB and its role in TB susceptibility. While studies of this gene have
had mixed results, all the studies, including my own, have had small
sample sizes. In the first study of \emph{in vitro} differences in
gene expression between susceptible and resistant individuals,
\emph{CCL1} was found to be differentially expressed based on
susceptibility status \citep{Thuong2008}. Furthermore, the same study
found that SNPs nearby \emph{CCL1}were associated with TB
susceptibility in an independent cohort. In Chapter \ref{ch:tb}, I
found that \emph{CCL1} was one of the genes which changed expression
level specifically in response to infection with mycobacteria. In
Chapter \ref{ch:tb-suscept}, I found that \emph{CCL1} was one of two
genes which had an effect size greater than 2 between susceptible and
resistant individuals in the noninfected state and also a p-value less
than 0.01 in two GWAS studies of TB susceptibility. There were many
differences between these studies (e.g. cell type, ethnicity of
donors, timepoints RNA was collected post-infection), yet \emph{CCL1}
was still a top hit in all three studies. I believe this warrants
further investigation. As an example, one could you CRISPR/Cas to
modify individual SNPs in THP-1 cells (a common cell line model of
monocytes) and test for differences in the response to MTB
infection. Another idea, since \emph{CCL1} is a secreted chemokine,
would be to add varying amounts of exogenous \emph{CCL1} to the
\emph{in vitro} system to detect an effect on the innate immune
response.

In Chapter \ref{ch:singleCellSeq}, I described my work on single cell
RNA sequencing (scRNA-seq) \citep{Tung2016}. scRNA-seq is a relatively
new technique that enables the investigation of gene regulatory
changes at a much finer resolution than the bulk RNA-seq projects I
performed in Chapters \ref{ch:tb} and \ref{ch:tb-suscept}. While this
new technology is exciting, we must exercise the same caution as when
performing any large-scale genomics experiment. Early studies of the
Fluidigm C1 system for scRNA-seq that focused on the technical sources
of variation largely focused on the variation from well to well within
just one C1 chip; whereas, the studies investigating biological
phenomena tended to use multiple C1 chips without addressing the
obvious confounding batch effects (this problem is nicely highlighted
by \citep{Hicks2015}). Before conducting large scale scRNA-seq
experiments, we first aimed to better understand the technical factors
affecting the design of such studies. To do so, we performed scRNA-seq
of three C1 chip replicates of three HapMap Yoruba individuals.

From these data, we learned many important lessons. First, by
performing subsampling analyses, we determined that sequencing
approximately 1.5 million reads for at least 75 cells from a given
individual was sufficient for detecting most expressed genes,
achieving a high correlation between the sum of the gene expression
levels across the single cells and the gene expression levels from
bulk sequencing of 10,000 cells, and achieving a high correlation
between the cell-to-cell variance in the gene expression levels across
the subset of single cells and the cell-to-cell variance measured in
all the single cells we collected for an individual (which ranged from
142 to 221). Second, we observed technical variation introduced from
the processing of the C1 batches. While this was expected, we also
observed unexpected aspects of this batch effect. The ERCC spike-in
controls which were added to each well and could potentially be used
to correct for this effect across C1 chips was affected not only by
technical variation but also by the biological variation (differences
between individuals). This entanglement of the technical and
biological sources of variation renders the spike-ins insufficient for
modeling technical variation between C1 chips (however they can still
be used to model technical variation between wells of the same C1
chip). This confounder occurred despite our use of unique molecular
identifiers (UMIs) to account for the bias introduced by amplifying
RNA from a small original source of just one cell. In fact, we found
that the conversion of reads to unique molecules was affected by
inter-individual differences. Third, even with our small sample size
of only three individuals, we were able to identify inter-individual
differences in the cell-to-cell gene expression variance, or gene
expression noise. This lends further support to the notion that gene
expression noise is a relevant factor that can affect biological
processes. Fourth, we demonstrated that we can use the single
nucleotide polymorphisms (SNPs) present in the RNA-seq reads to
identify the individual of origin for a given single cell. This
enables us to use a crossed-design where single cells from multiple
individuals are included on the same C1 chip and later each well is
assigned to each individual based on the RNA-seq reads obtained. Our
initial nested design was inefficient because we collected hundreds of
single cells per individual across the multiple technical
replicates. From our subsampling we knew that collecting 75 high
quality single cells was sufficient. With a crossed design, we can
obtain about one C1 chip worth of wells (96) while still modeling the
technical variation across C1 chips.

Given the promising results from our first study, our next study will
aim to further investigate the impact of genetic variation on gene
expression noise by measuring single cell gene expression levels in 60
individuals. The design of the study is informed by our previous
findings. First, we will put single cells from multiple individuals on
each C1 chip because we know we can identify the individual based on
the RNA-seq reads. Second, we will repeat each individual across C1
chips until they obtain on average 96 wells (e.g. one C1 chip) because
this will get us close to our target of 75 single cells after removing
low quality cells. Third, we will replace the ERCC spike-ins with RNA
from a distantly related model organism. With many more technical
spike-ins gene to measure, these will be more useful for modeling
technical variation \citep{Risso2014}. Using this study design, weâ€™ll
be able to efficiently measure gene expression noise from many
individuals while still properly accounting for technical variation.

Returning to the \emph{in vitro} models of bacterial infection from my
other studies, I can imagine future single cell studies that shed
further light on the innate immune response. While we infect the cells
at a multiplicity of infection of 1:1, some cells will still be
infected by multiple bacteria and others not infected at
all. Furthermore, there could be variation in this distribution of the
number of bacteria per cell across individuals. In order to
efficiently measure single cell gene expression in response to
infection, I would put uninfected cells from one individual on the
same C1 chip as the infected cells from a different individual. Also,
since the MTB H37Rv strain we typically use has a GFP tag, we could
use high-throughput fluorescence microscopy of each well to count the
number of bacteria per cell. With this high resolution data, we could
differentiate between inter-individual differences in the innate
immune response due to differences in the number of infected cells (or
the number of bacteria per cell) or differences in the innate immune
response in the infected cells.

A common theme of all my projects is accounting for technical
biases. Although only Chapter \ref{ch:singleCellSeq} has a main focus
on mitigating batch effects, all my projects required close attention
to this problem. This is because all genomics studies need to account
for batch effects in both the design and analysis of the data,
otherwise the results are meaningless. There will signal in any large
data set, but it will only inform biological insight if the signal
arises from the biological processes being studied.

In Chapter \ref{ch:tb}, we collected a total of 156 RNA-seq
samples. During the batch processing, we ensured that the biological
factors of interest (bacterial infection, timepoint, individual) were
balanced to avoid introducing spurious signal. Furthermore, upon data
exploration, we observed that the processing batch and the RNA quality
score (RIN) were correlated with the first principal component. After
regressing these two variables, the first principal component was the
effect of timepoint and the second PC was the effect of
infection. Importantly, we obtained similar results whether we
protected the variables of interest in the linear model or not when
regressing the technical variables. This was a result of the careful
planning of the batch processing to avoid confounding biological and
technical variables.

In Chapter \ref{ch:tb-suscept}, I once again designed the batch
processing to balance the biological factors of interest
(susceptibility status, treatment, individual). Conveniently, this
project did not have large scale batch effects, likely due to the
smaller overall sample size of 48. However, accounting for a batch
effect was critical for training a classifier on the current data set
and testing it on an independent data set. Not only were the studies
performed years apart, but the gene expression levels were measured
using different technologies. Thus I was only able to compare the two
studies after normalizing each sample and removing the large batch
effect by regressing the first principal component of the combined
data set. Testing the classifier without accounting for the batch
effect would have given poor results simply due to technical reasons.

In Chapter \ref{ch:singleCellSeq}, one of the main motivations for the
study was understanding the magnitude of the batch effect of
collecting scRNA-seq on separate C1 chips. While the technical effect
of C1 batch was smaller in magnitude compared to the biological effect
of individual (assessed using variance components analysis), not
including technical replicates would attribute the substantial
technical effect to the biological effect. Just as we require
replication for established genomic protocols, it is also necessary to
replicate scRNA-seq experiments, especially since the standard ERCC
spike-ins appear to be affected by both biological and technical
factors. Fortunately, we were able to devise a strategy to reduce the
required number of C1 replicates by combining single cells from
multiple individuals onto each C1 chip and then replicating the
multiple individuals across multiple chips. This crossed design
accounts for batch effects while minimizing the required replication.

In summary, technical batch effects need to be considered from the
initial design of a genomics experiments through to the data analysis
and interpretation of the results.

I have identified hundreds of genes involved in fighting MTB
infections.  More broadly, I have demonstrated that a joint Bayesian
model is an effective tool for analyzing the large-scale multivariate
data produced by genomic studies. Lastly, I have determined an
effective study design for future single cell studies.
